toolkits/global/packages/global-layout-cluster/scss/50-components/layout-cluster.scss

```diff
@@ -0,0 +1,36 @@
+.l-cluster {
```

Comment:
Is there a reason for using `l-`? Our convention for components is `c-`

Reply:
There was some [previous discussion around this](https://github.com/springernature/frontend-toolkits/pull/577#issuecomment-927993134).

I guess something like the front-end playbook would need an update to classify this more?

Unlike component things like a button, or a header - the `l-` layout primitives would be something added to the `c-` component to avoid over duplication of CSS (or it could used on its own). And like margin, or padding, it's not really a `u-` utility as they're, generally, doing more than one thing in the CSS.

There is now precedent with `l-stack`, `l-with-sidebar`, and now `l-cluster` in the front-end toolkits. But maybe a retrospective front-end open space ticket might be useful?

These layouts are based on @Heydon and [Andys](http://twitter.com/piccalilli_) [Every Layout](https://every-layout.dev).

Reply:
@alexkilgour Yep, basically these don't represent lexical components, more like lower-level helpers. Something between utilities and components. Hope that makes sense.